### DISCUSSION TITLE
Error on W3C validator using the Toggle and Accordion widget.

### DISCUSSION THREAD

**Discussion by BimberOnline** [elementor/elementor#23208] [Developers Community]
When using the toggle or accordion widget, the [validator W3C](https://validator.w3.org/nu/?showsource=yes&showoutline=yes&doc=https%3A%2F%2Fbimberonline.com%2Fever-organize%2Ftemplate-kit%2Ffaq) generates an error message:

**The "a" element must not appear as a descendant of an element with the attribute role="button".**

Here is a sample link to consult on the validator:
[https://bimberonline.com/ever-organize/template-kit/faq]

We have attempted the following procedures without success:

- Elementor > Settings > Advanced > CSS Print Method > Internal Embedding
- Elementor > Tools > Regenerate CSS & Data
- Clearing browser and server cache
- Disabled all plugins except Elementor and Elementor Pro.
- Used the Hello Elementor theme.

I have searched for a solution to this error but have not found one. Any clarification on resolving this would be greatly appreciated.

**Comment by emanuel-gold** [comment]
I am currently dealing with the same issue as this is a red flag for WCAG compliance. Any help would be appreciated.
